packages/server/src/helpers/sanitize-address-middleware.js

```diff
@@ -0,0 +1,14 @@
+import { isAddress } from 'web3-utils'
+
+const sanitizeAddresses = object => {
+  Object.keys(object).forEach(key => {
+    if (!!object[key] && typeof object[key] === 'object') sanitizeAddresses(object[key])
+    else if(isAddress(object[key])) object[key] = object[key].toLowerCase()
```

Comment:
I would do a stricter check here—let's transform to checksum and error if that fails before passing it through as all lowercased.

Reply:
Ah, actually I think `isAdress()` checks for the checksum already right? In that case, ignore me :).

Reply:
yes it does :)